src/Microsoft.AspNetCore.Sockets.Abstractions/Features/ITransferModeFeature.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Sockets.Abstractions.Features
```

Comment:
Fix the namespace - all other features are in Microsoft.AspNetCore.Sockets.Features